include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -81,8 +84,17 @@ namespace aspect
 
             Point<3> push_forward_ellipsoid (const Point<3> &phi_theta_d, const double semi_major_axis_a, const double eccentricity) const;
             Point<3> pull_back_ellipsoid (const Point<3> &x, const double semi_major_axis_a, const double eccentricity) const;
+            Point<3> push_forward_topography (const Point<3> &phi_theta_d_hat) const;
```

Comment:
Please add some documentation to the functions. Even if the state of the rest of this class is a slight mess, no need to continue that tradition. I would be especially interested in which function gets the mapped coordinates and returns the original coordinates and the other way around (or do these functions something entirely different?).